python/pyspark/mllib/feature.py

```diff
@@ -123,20 +132,6 @@ class StandardScalerModel(JavaVectorTransformer):
 
     Represents a StandardScaler model that can transform vectors.
     """
-    def transform(self, vector):
-        """
-        Applies standardization transformation on a vector.
-
-        Note: In Python, transform cannot currently be used within
-              an RDD transformation or action.
-              Call transform directly on the RDD instead.
-
-        :param vector: Vector or RDD of Vector to be standardized.
-        :return: Standardized vector. If the variance of a column is
```

Comment:
This part is specific to this transformer.  Can you please add it somewhere in the doc for StandardScalerModel?